mentoring-wg/contributor-growth.md

```diff
@@ -0,0 +1,12 @@
+CNCF TAG Contributor Strategy
+# Mentoring Working Group
+
+
+## About TAGCS Mentorship Working Group
```

Comment:
It feels weird to keep repeating this header section in every single document -- especially the documents that don't yet have any content.  Can you explain the rationale, here?

Reply:
@jaytiaki @Riaankl

Reply:
That file seem out of place.
Deleted it in https://github.com/cncf/mentoring/pull/925
@nate-double-u @jaytiaki @jberkus